### DISCUSSION TITLE
posts/front-end-developer-job-search-review/

### DISCUSSION THREAD

**Discussion by giscus** [benkim077/blog#58] [Announcements]
# posts/front-end-developer-job-search-review/

이젠 어떻게 살아야 할까?

https://benkim077.github.io/posts/front-end-developer-job-search-review/

**Comment by Byongho96** [comment]
꼭 목표한 바 이루시길 바래요~~!! 그나저나 프레임워크 없이 만든 블로그 멋있네용

**Reply by benkim077** [reply]
병호님 응원 댓글 감사해요~ 😊

프레임워크 없이 개발하면서 불편한 부분도 있었지만, 필요한 부분을 찾아가면서 개발했던 점이 참 재밌었던 것 같아요. 나중에 SSG 프레임워크로 마이그레이션 하는 것도 재밌을 것 같아요 ㅋㅋ
